src/DepressurizerCore/Models/DatabaseEntry.cs

```diff
@@ -650,6 +650,7 @@ private AppType ScrapeStoreHelper(int id)
             HttpWebResponse resp = null;
             try
             {
+                LastStoreScrape = DateTimeOffset.Now.ToUnixTimeSeconds();
```

Comment:
This should be outside the try-catch. If the scraping fails then we don't want to update the LastStoreScrape as we would keep the current data, which can be nothing or old, and it won't update until the time difff is too big again.

Reply:
For me this fix work better then the version you provide at the moment. This will mostly destroy the categoryzen of all user who use your tool. I use a lot of own categoryzen and for me my dirty quickfix work. And I must admit that I don't understand how some things work in this project. And their are a lot of side effects...

I can understand your concern. Their are some secondary effect when the tag is not set.
*Games loose their category when LastStoreScrape not set
*Depressurizer ask every time to rescrape the games without time tag when you use the AutoCategorysen Button.
**This problem are not a problem with your versions the 0.7.x hide it with the provide of a database out of the box. I delate the database who is provided and get the same problems as here.